services/workflows-service/src/workflow-defintion/workflow-definition.repository.ts

```diff
@@ -6,6 +6,7 @@ import { GetWorkflowDefinitionListDto } from '@/workflow-defintion/dtos/get-work
 import { validateDefinitionLogic } from '@ballerine/workflow-core';
 import { Injectable } from '@nestjs/common';
 import { Prisma, PrismaClient, WorkflowDefinition } from '@prisma/client';
+import { createDemoWorkflow } from '../../prisma/data-migrations/shared/create-demo-workflow';
```

Comment:
To remain compatible with the OSS (that don't have access to the `data-migrations` folder), I think it might be better to have this file in the `workflows-service`